### ISSUE TITLE
Feature request: Allow exclude_file to be nonexisting

### ISSUE THREAD

**Issue by fatbasstard** [GrantBirki/json-yaml-validate#67]
Hi!

I'm using this validator in a "centralized" solution where we've got [reusable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows). 

In this setup I now want to make it "configurable" by allowing the calling repository to have a `exclude_file` if needed. 

In the reusable workflow I now specifiek a standard file name with the idea: If you want to exclude files in your repository, just add an exclude file.

Did this in this way:

```yaml
      - name: json-yaml-validate
        uses: GrantBirki/json-yaml-validate@v3.0.0
        with:
          exclude_file: ".staticignore" # gitignore style file that contains a list of files to exclude
          use_gitignore: "false"
```

I was hoping that it would just ignore (or at least continue) if the file doesn't exist. This would make my static file check "configurable". But I get the following error now:

```Run GrantBirki/json-yaml-validate@v3.0.0
Error: ENOENT: no such file or directory, open '.staticignore'
    at readFileSync (node:fs:448:[20](https://github.com/xxx/xxx/actions/runs/9697194409/job/26760772582?pr=431#step:3:21))
    at new Exclude (/home/runner/work/_actions/GrantBirki/json-yaml-validate/v3.0.0/src/functions/exclude.js:16:1)
    at run (/home/runner/work/_actions/GrantBirki/json-yaml-validate/v3.0.0/src/main.js:7:1)
    at /home/runner/work/_actions/GrantBirki/json-yaml-validate/v3.0.0/src/main.js:14:1
    at /home/runner/work/_actions/GrantBirki/json-yaml-validate/v3.0.0/dist/index.js:69289:3
    at Object.<anonymous> (/home/runner/work/_actions/GrantBirki/json-yaml-validate/v3.0.0/dist/index.js:69292:12)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Function.Module._load (node:internal/modules/cjs/loader:10[24](https://github.com/xxx/xxx/actions/runs/9697194409/job/26760772582?pr=431#step:3:25):12)```
    

Can this exclude_file be made in a way that is allows a non existing file to be specified?

**Comment by GrantBirki** [comment]
@fatbasstard This is an excellent idea! I have this on my radar. I'll try my best to get this implemented for you in the near future. I am traveling for the next week or so and will take a stab at this when I get back. 🙇

**Comment by GrantBirki** [comment]
@fatbasstard I have published a new release ([`v3.1.0`](https://github.com/GrantBirki/json-yaml-validate/releases/tag/v3.1.0)) and also updated the main `v3` tag with these changes. Please go ahead and try it out and let me know how it goes! 🚀

**Comment by fatbasstard** [comment]
Works like a charm! Thanks a lot 🎉
